docs/src/xhtml/components/tags/tag.xhtml

```diff
@@ -53,6 +53,11 @@
 									"type": "boolean",
 									"desc": "Tag is deletable? (only visually)"
 								},
+								{
+									"name": "doremove",
+									"type": "boolean",
+									"desc": "Remove the element, when you set it true, default is false"
```

Comment:
I think it's not right! The default value is true.

Reply:
Yes, you are right